src/environment.ts

```diff
@@ -6,24 +6,30 @@ import Constants from 'expo-constants';
  * the env var values. This can then be returned via the getEnvVars function.
  */
 const ENV = {
-    dev: {
-        apolloURI: 'localhost:4000/graphql',
-    },
-    staging: {}     // sample object for a 'staging' release channel
+  dev: {
+    apolloURI: 'localhost:4000/graphql',
```

Comment:
What do yall think about using [dotenv here instead](https://docs.expo.io/guides/environment-variables/#using-a-dotenv-file) in order to move environment variables into its separate file? I think it's better to hide config information in a separate `.env` file but other people might have different opinions.

Reply:
I think in this case it's fine to have the development URI/port within the JS file since that way it stays consistent across instances of the app + devs wouldn't have to setup a whole `.env` file for that one config. Though if future configs are planned then it might be worth considering having a separate `.env` file.

Reply:
I think in this case it's fine to have the development URI/port within the JS file since that way it stays consistent across instances of the app + devs wouldn't have to setup a whole `.env` file for that one config. Though if future configs are planned then it might be worth considering having a separate `.env` file.

Reply:
I'm guessing we'll have more configs down the line, but yeah I see your point

Reply:
@thaigillespie that's a great point man. It's the best practice on a very large+complex project. That said, we can temporarily avoid mental overhead by keeping it as simple as we can – so long as the complexity of the project doesn't demand a separate `.env` file(s). Though, I'm still down to try anything.

Reply:
Apollo actually won't be necessary anymore (see below), so I'll take this out. Future env vars will definitely be needed (API endpoint, YCW Site endpoint), so when the time comes, dotenv seems to be the better solution vs this strategy. 

Comment on Apollo: Amplify generates the queries + mutations for us that we can then use via src/graphql and state management will be done via React Context.